che.sh

```diff
@@ -740,38 +740,66 @@ update_che_image() {
 
 execute_che_mount() {
   debug $FUNCNAME
-  if [ ! $# -eq 3 ]; then 
-    error "${CHE_MINI_PRODUCT_NAME} mount: Wrong number of arguments provided."
-    return
-  fi
 
-  MOUNT_PATH=$(get_mount_path "${2}")
+  # Determine the mount path to do the mount
+  info "${CHE_MINI_PRODUCT_NAME} mount: Setting local mount path to ${PWD}"
+  MOUNT_PATH=$(get_mount_path "${PWD}")
+  HOME_PATH=$(get_mount_path "${HOME}")
 
-  if [ ! -e "${MOUNT_PATH}" ]; then
-    error "${CHE_MINI_PRODUCT_NAME} mount: Path provided does not exist."
-    return
-  fi
+  # If extra parameter provided, then this is the port to connect to
+  if [ $# -eq 1 ]; then
+    info "${CHE_MINI_PRODUCT_NAME} mount: Connecting to remote workspace on port ${1}"
+    WS_PORT=${1}
 
-  if [ ! -d "${MOUNT_PATH}" ]; then
-    error "${CHE_MINI_PRODUCT_NAME} mount: Path provided is not a valid directory."
-    return
+  # Port not provided, let's do a simple discovery of running workspaces
+  else 
+    info "${CHE_MINI_PRODUCT_NAME} mount: Searching for running workspaces with open SSH port..."
+
+    CURRENT_WS_INSTANCES=$(docker ps -aq --filter "name=_che_dev-machine")
+    CURRENT_WS_COUNT=$(echo $CURRENT_WS_INSTANCES | wc -w)
+    
+    if [ $CURRENT_WS_COUNT -eq 0 ]; then
+      error "${CHE_MINI_PRODUCT_NAME} mount: We could not find any running workspaces"
+
+    elif [ $CURRENT_WS_COUNT -eq 1 ]; then
+      RUNNING_WS_PORT=$(docker inspect --format='{{ (index (index .NetworkSettings.Ports "22/tcp") 0).HostPort }}' ${CURRENT_WS_INSTANCES})
+      info "${CHE_MINI_PRODUCT_NAME} mount: Connecting to remote workspace on port $RUNNING_WS_PORT"
```

Comment:
side note
if we're always using info "${CHE_MINI_PRODUCT_NAME} 
"${CHE_MINI_PRODUCT_NAME} " should be inside info function

or we need to add another info commands to avoid to write ${CHE_MINI_PRODUCT_NAME}  in all cals